rover/source/docs/manual-mode.rst

```diff
@@ -4,8 +4,9 @@
 Manual mode
 ===========
 
-In Manual mode the transmitter's steering and throttle sticks directly control the vehicle's throttle and steering output.
+In Manual mode, the transmitter's steering and throttle sticks directly control the vehicle's throttle and steering output.
 
-- this mode does not require a position estimate (i.e. GPS is not required)
-- it is recommended to always have one :ref:`transmitter mode switch <common-rc-transmitter-flight-mode-configuration>` position assigned to this mode
-- the throttle slew still applies. See the bottom of the :ref:`Tuning Speed and Throttle <rover-tuning-throttle-and-speed>` page for more details of the throttle slew limit
+- This mode does not require a position estimate (i.e. GPS is not required).
+- It is recommended to always have one :ref:`transmitter mode switch <common-rc-transmitter-flight-mode-configuration>` position assigned to this mode.
+- The throttle slew still applies. See the bottom of the :ref:`Tuning Speed and Throttle <rover-tuning-throttle-and-speed>` page for more details about the throttle slew limit.
+- Steering angle speed scaling can be enabled via :ref:`MANUAL_OPTIONS<MANUAL_OPTIONS>`
```

Comment:
how can speed scaling be applied if there is no GPS, which is not required in this mode?

Reply:
It's just a change of a bool. The scaling fails to apply if ground speed is not present just as it would in modes that enable steering speed scaling by default.